features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -162,7 +162,7 @@ export const ExplorerProvider = ({ children }) => {
     }
 
     function entityJurisdictionByPpid(ppid) {
-        return entityObjectByPpid(ppid).jurisdiction;
+        return entityObjectByPpid(ppid)?.jurisdiction;
```

Comment:
Necessary due to data inconsistencies. Will solve it in https://jira.polypoly.eu/browse/PROD4POD-1210

Reply:
I think this might flunk when #584 lands. It will return "undefined". Is the client of the function ready to deal with that?

Reply:
It will only return undefined when there are data inconsistencies. Which there are right now. It will just add a jurisdiction called "undefined" in that case